index.html

```diff
@@ -44,7 +44,10 @@ <h2>Lambda School Students: "We're the best!"</h2>
         naboo ackbar tatooine. Hutt lars padmé darth. Maul solo darth darth jabba qui-gon chewbacca darth maul. Moff baba wicket
         han. C-3po antilles moff qui-gon ahsoka aayla dooku amidala. Palpatine droid amidala droid k-3po twi'lek padmé wookiee. Leia
         moff calamari mon obi-wan. Solo grievous lando coruscant. Jinn darth palpatine obi-wan mon.</p>
-        <span class='expandButton'></span>
+          <div class= "article-buttons">
```

Comment:
The solution I provided earlier will allow you to remove these extra hidden buttons and keep the original button that was given